test/compare_output.zig

```diff
@@ -444,4 +444,18 @@ pub fn addCases(cases: &tests.CompareOutputContext) {
 
         tc
     });
+
+    cases.add("bad dll name on windows",
```

Comment:
can you make this a behavior test? look at test/behavior.zig and test/cases/*

generally we prefer these kind of tests when possible because they run much faster than compare_output tests.

Reply:
better yet, add this test somewhere in std/  because that's what it's testing.
behavior tests are for language features.